CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Flexy File Utility Library Changelog
+
+## Version 1.1
+### Features
+- Added function for appending strings to existing files
+### Bug Fixes
+- N/A
+### Other
+- N/A
+
+## Version 1.0
+### Features
+- Initial Library Release
```

Comment:
why did you make this line all caps where line 5 is not?

Reply:
This was a copy/paste and adapt from the Thingworx readme. I've updated so it is uniform.